### PR TITLE
Override django-registration RegisterView to emit custom orchestra signal

### PR DESCRIPTION
**Pull request by joshblum**

### CODE REVIEW

File: orchestra/signals.py [hunk 1]
```diff
@@ -1,9 +1,12 @@
 from django.dispatch import receiver
-from registration.signals import user_registered
+from django.dispatch import Signal
 
 from orchestra.models import Worker
 
+# A user has activated his or her account.
+orchestra_user_registered = Signal(providing_args=["user", "request"])
```
Comment (marcua) [inline orchestra/signals.py]: single-quotes

File: orchestra/views.py [hunk 2]
```diff
@@ -187,3 +193,49 @@ def submit_task_assignment(request):
 # A simple status endpoint for things like health checks, etc.
 def status(request):
     return HttpResponse('OK')
+
+
+class OrchestraRegistrationView(RegistrationView):
```
Comment (marcua) [inline orchestra/views.py:198]: is this impl essentially the same as RegistrationView except we do `signals.orchestra_user_registered.send` at the end?  if so, can we make our implementation call the parent class' implementation?

Comment (joshblum) [inline orchestra/views.py:198]: I wanted to, but the parent class emits the signal at the end of it. Would this be a problem if another project was using registration package and listening to the signal but didn't want to get notified by orchestra users?

Comment (marcua) [inline orchestra/views.py:198]: ok cool---let's leave as you have it :)

**Comment by marcua** [comment]
lgtm assuming you've manually tested that registration still works

**Comment by joshblum** [comment]
screen shot 2016-02-24 at 11 22 48
 yallah :)

**Pull request merged**